src/screens/VaultScreen.js

```diff
@@ -13,11 +14,305 @@ export default class VaultScreen extends React.Component {
     ),
   };
 
+  async componentDidMount() {
+    await Font.loadAsync({
+      'source-sans-pro-black': require('../.././assets/fonts/SourceSansPro-Black.ttf'),
+      'source-sans-pro-black-italic': require('../.././assets/fonts/SourceSansPro-BlackItalic.ttf'),
+      'source-sans-pro-bold': require('../.././assets/fonts/SourceSansPro-Bold.ttf'),
+      'source-sans-pro-bold-italic': require('../.././assets/fonts/SourceSansPro-BoldItalic.ttf'),
+      'source-sans-pro-extra-light': require('../.././assets/fonts/SourceSansPro-ExtraLight.ttf'),
+      'source-sans-pro-extra-light-italic': require('../.././assets/fonts/SourceSansPro-ExtraLightItalic.ttf'),
+      'source-sans-pro-italic': require('../.././assets/fonts/SourceSansPro-Italic.ttf'),
+      'source-sans-pro-light': require('../.././assets/fonts/SourceSansPro-Light.ttf'),
+      'source-sans-pro-light-italic': require('../.././assets/fonts/SourceSansPro-LightItalic.ttf'),
+      'source-sans-pro-regular': require('../.././assets/fonts/SourceSansPro-Regular.ttf'),
+      'source-sans-pro-semibold': require('../.././assets/fonts/SourceSansPro-SemiBold.ttf'),
+      'source-sans-pro-semibold-italic': require('../.././assets/fonts/SourceSansPro-SemiBoldItalic.ttf'),
+    });
+    this.setState({fontLoaded: true});
+  }
+  constructor(props) {
+    super(props);
+    this.state = {
+      titleName: "Resources",
+      searchText: '',
+      filter: [],
+      names: [
+        {name: 'CLEAR', selected:false, id: 1},
```

Comment:
Is clear just a button that clears the selected state of the other buttons? If that's the case then idt it needs a state.